src/sections/CTASection/CTASection.tsx

```diff
@@ -20,8 +21,9 @@ const HighlightPanel = styled.div`
 const ContentWrapper = styled.div`
   display: flex;
   align-items: center;
-  gap: ${rem('100px')};
-  padding: 40px;
+  justify-content: space-between;
+  padding: 80px 40px;
+  gap: ${rem('50px')};
```

Comment:
just for the future - we can simply pass the number to the `rem` helper function, for example:

```suggestion
  gap: ${rem(50)};
```

Nothing blocking, as we use it this way everywhere, just saying.